src/test/java/com/tvtracker/enterprise/EnterpriseApplicationTests.java

```diff
@@ -103,20 +102,19 @@ private void givenUserHasCreatedAMediaEntry() throws Exception {
     private int whenUserUpdatesMediaEntry() throws Exception {
         List<MediaEntry> mediaEntries = mediaEntryService.fetchMediaEntriesByUsername(TEST_USERNAME);
         // update a media entry
-        MediaEntry mediaEntry = mediaEntries.get(0);
-        MediaEntry newMediaEntry = new MediaEntry();
-        newMediaEntry.setEntryId(mediaEntry.getEntryId());
-        newMediaEntry.setUsername(mediaEntry.getUsername());
-        newMediaEntry.setTitle(mediaEntry.getTitle());
-        newMediaEntry.setType(mediaEntry.getType());
-        newMediaEntry.setDescription("Terrible");
-        newMediaEntry.setPlatform("Hulu");
-        newMediaEntry.setWatched(true);
-
-        boolean success = mediaEntryService.updateMediaEntry(newMediaEntry);
+        mediaEntry = mediaEntries.get(0);
+        mediaEntry.setEntryId(mediaEntry.getEntryId());
+        mediaEntry.setUsername(mediaEntry.getUsername());
+        mediaEntry.setTitle(mediaEntry.getTitle());
+        mediaEntry.setType(mediaEntry.getType());
+        mediaEntry.setDescription("Terrible");
+        mediaEntry.setPlatform("Hulu");
+        mediaEntry.setWatched(true);
+
+        boolean success = mediaEntryService.updateMediaEntry(mediaEntry);
```

Comment:
No need to create a new object as you can just use the setter methods to update the properties of an already existing object. This will eliminate clutter which in term lessens technical debt.
https://martinfowler.com/bliki/TechnicalDebt.html

Reply:
Careful on this... what if you don't call every setter method?  You could end up with some data leftover from the prevoius object.  This is easy to do by mistake.

Reply:
Very true I didn't even think of that!